src/jdk.jshell/share/classes/jdk/jshell/SourceCodeAnalysisImpl.java

```diff
@@ -949,6 +1024,17 @@ private Predicate<Element> createAccessibilityFilter(AnalyzeTask at, TreePath tp
         };
     }
 
+    private boolean isAnnotationAttributeValueType(AnalyzeTask at, TypeMirror type) {
```

Comment:
This name is misleading - it is checking if a static final field type is suitable for an annotation attribute.  This is more restrictive than general annotation element return types, which also allows annotation interfaces and one-dimensional arrays of primitives, enums, strings, and annotation interfaces.